### ISSUE TITLE
Dependencies installation does not work in a proper way

### ISSUE THREAD

**Issue by defra91** [Automattic/kue#874]

**Comment by behrad** [comment]
Have you any shared dependencies with Kue's package.json in your project?

**Comment by defra91** [comment]
I have two shared dev dependencies:

- `async`
- `chai`

**Comment by behrad** [comment]
I think if their versions match Kue's one, they won't be installed inside kue, not to be duplicated. I'm not a NPM expert, you can ask npm guys about that :)

**Comment by defra91** [comment]
Yes that would make sense for those two libraries, but I actually have several which are not declared in my `package.json`. I will post this issue in a more npm related section and if I will report the solution.
